client/app/views/popover_screens/duplicate.coffee

```diff
@@ -0,0 +1,38 @@
+PopoverScreenView = require 'lib/popover_screen_view'
+
+module.exports = class ConfirmDuplicatePopoverScreen extends PopoverScreenView
```

Comment:
Ok maybe I misunderstood its real role. If you confirm your choice, it's ok for me (and I love long self-describing names).